src/main/java/org/kohsuke/github/GHUser.java

```diff
@@ -40,7 +40,7 @@ public class GHUser extends GHPerson {
     protected String ldap_dn;
 
     /** The suspended_at */
-    private String suspendedAt;
+    private String suspended_at;
```

Comment:
Don't need this. Jackson does this for you.

Reply:
👌  Rollbacked to `suspendedAt`.